pkg/config/config.go

```diff
@@ -1,68 +1,74 @@
-package main
+package config
 
 import (
 	"errors"
 
 	"github.com/conductorone/baton-sdk/pkg/field"
-	"github.com/spf13/viper"
 )
 
 var (
 	TokenField = field.StringField(
 		"token",
-		field.WithRequired(true),
```

Comment:
why is required removed?

Reply:
@laurenleach I can add it back in. For some reason, I had a memory of you mentioning not using this. But maybe that was when `WithRequired` was `False`?? Now I can't remember which connector it was anyway.

Reply:
Oh that for baton-github, your PR had added `WithRequired(true)`, but Ben had removed it since he added a new auth method. We need to keep required when it's a field that is always required